src/test/java/testCases/Validate_CashierPassword.java

```diff
@@ -0,0 +1,157 @@
+//Test case to check error validation of each field on the Financial Assessment page
+package testCases;
+
+import org.testng.annotations.Test;
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import pageObjects.CashierPassword_Page;
+import pageObjects.Cashier_Page;
+import pageObjects.MainAccount_Menu;
+import pageObjects.MainMenu_Tab;
+import pageObjects.Security_Page;
+import utility.Constant;
+import appModules.CashierPassword_Action;
+
+import appModules.Login_Action;
+
+public class Validate_CashierPassword {
+    public WebDriver driver;
+    //Test Method to login into the site
+    @Test(priority=0)
+    public void Login() {
+        Login_Action.Execute(driver,Constant.Email,Constant.Password);
+    } 
+    //Test Method to navigate to cashier password page
+    @Test(priority=1)
+    public void NavigateToCashierPasswordPage() {
+        MainAccount_Menu.link_MainAccount(driver).click();
+        MainAccount_Menu.link_Security(driver).click();
+        Security_Page.cashierPassword_link(driver).click();
+    }
+    //Check cashier password page is load properly 
+    @Test(priority=2)
+    public void CheckPageLoad() {
+        Assert.assertTrue(CashierPassword_Page.title_page(driver).isDisplayed());
+        Assert.assertTrue(CashierPassword_Page.title_form(driver).isDisplayed());
+        CashierPassword_Action.checkTitle(driver);
+    }
+    //Check empty field validation
+    @Test(priority=3)
+    public void check_emptyField() {
+        CashierPassword_Page.cashierPassword(driver).clear();
+        CashierPassword_Page.cashierPassword(driver).sendKeys("");
+        CashierPassword_Page.confirmPassword(driver).clear();
+        CashierPassword_Page.confirmPassword(driver).sendKeys("");
+        CashierPassword_Page.updateButton(driver).click();
```

Comment:
Any specific reason for not using CashierPassword_Action methods to enter into fields and submitting form in each of the tests ?